torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -11,3 +13,26 @@ def __call__(self, *, kernel, grid, args, kwargs):
 
 
 triton_kernel_wrapper_mutation = TritonKernelWrapperMutation()
+
+
+# Used for wrapping a Triton Kernel in a functional manner
+class TritonKernelWrapperFunctional(HigherOrderOperator):
+    def __init__(self):
+        super().__init__("triton_kernel_wrapper_functional")
+
+    def __call__(self, *, kernel, grid, args, kwargs):
```

Comment:
Consider normalizing the args when we create the graph so we don't need both args and kwargs.